### PR TITLE
Clarify refresh parameter in the `_bulk` API

### PR DESCRIPTION
**Pull request by bleskes**
See #11690

### CODE REVIEW

**Comment by s1monw** [comment]
LGTM

File: docs/reference/docs/bulk.asciidoc [hunk 1]
```diff
@@ -167,7 +167,8 @@ The `refresh` parameter can be set to `true` in order to refresh the relevant
 primary and replica shards immediately after the bulk operation has occurred
 and make it searchable, instead of waiting for the normal refresh interval to
 expire. Setting it to `true` can trigger additional load, and may slow down
-indexing.
+indexing. Due to it's costly nature, the `refresh` paramter is set on the bulk request level
```
Comment (tlrx) [inline docs/reference/docs/bulk.asciidoc]: paramter -> parameter

**Comment by tlrx** [comment]
LGTM, minor typo

File: docs/reference/docs/bulk.asciidoc (hunk 1 above)
Comment (javanna) [inline docs/reference/docs/bulk.asciidoc]: s/it's/its

**Comment by bleskes** [comment]
@kevinkluge fyi - some automation in your name reopen this PR.. closing.

**Pull request closed without merging**